app/controllers/api/storage.php

```diff
@@ -1329,6 +1330,11 @@
             $file = Authorization::skip(fn() => $dbForProject->updateDocument('bucket_' . $bucket->getInternalId(), $fileId, $file));
         }
 
+        $deletes
+            ->setType(DELETE_TYPE_CACHE_BY_RESOURCE)
+            ->setResource('file/' . $file->getId())
+        ;
+
```

Comment:
deleting cache when updating a file makes sense. But why do we need to delete cache when creating a file ?